test/e2e/xtx/sender.go

```diff
@@ -0,0 +1,97 @@
+package xtx
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/txmgr"
+	"github.com/omni-network/omni/test/e2e/netman"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type Sender struct {
+	txMgr     txmgr.TxManager
+	portal    common.Address
+	abi       *abi.ABI
+	chainName string
+	chainID   uint64
```

Comment:
you can remove some fields here that are not used:
- chainI, chainName, abi, portal

Reply:
keeping them in for logging for now, until we can find root cause